templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/porter.yaml

```diff
@@ -1,10 +1,36 @@
 ---
 name: tre-service-guacamole-windowsvm
-version: 0.5.0
+version: 0.5.1
 description: "An Azure TRE User Resource Template for Guacamole (Windows 10)"
 dockerfile: Dockerfile.tmpl
 registry: azuretre
 
+custom:
+  # For information on vm_sizes and image_options, see README.me in the guacamole/user-resources folder
+  vm_sizes:
+    "2 CPU | 8GB RAM": Standard_D2s_v5
+    "4 CPU | 16GB RAM": Standard_D4s_v5
+    "8 CPU | 32GB RAM": Standard_D8s_v5
+    "16 CPU | 64GB RAM": Standard_D16s_v5
+  image_options:
+    "Windows 10":
+      source_image_reference:
+        publisher: MicrosoftWindowsDesktop
+        offer: windows-10
+        sku: 20h2-pro-g2
+        version: latest
+      conda_config: false
+    "Server 2019 Data Science VM":
+      source_image_reference:
+        publisher: microsoft-dsvm
+        offer: dsvm-win-2019
+        sku: server-2019
+        version: latest
+      conda_config: true
+    # "Custom Image From Gallery":
```

Comment:
Is this still needed here?

Reply:
I thought that a commented example might help with discovery (since the `source_image_name` option isn't used by either of the other options)?

Reply:
Yeah good point! Maybe have text that makes it a bit clearer, e.g. add a comment on top
`# Add a custom image definition here (see also link_to_docs.md)

Reply:
done!